src/v2/view-builder/views/phone/ChallengeAuthenticatorPhoneView.js

```diff
@@ -0,0 +1,126 @@
+import { loc, View, createCallout } from 'okta';
+import BaseView from '../../internals/BaseView';
+import BaseForm from '../../internals/BaseForm';
+import BaseFooter from '../../internals/BaseFooter';
+import BaseFactorView from '../shared/BaseFactorView';
+import { addSwitchAuthenticatorLink } from '../../utils/AuthenticatorUtil';
+
+const SHOW_RESEND_TIMEOUT = 30000;
+
+const ResendView = View.extend(
+  {
+    // To be shown after a timeout
+    className: 'phone-authenticator-challenge__resend-warning hide',
+    events: {
+      'click a.resend-link': 'handleResendLink'
+    },
+
+    initialize () {
+      const resendText = (this.model.get('mode') === 'sms')
+        ? loc('oie.phone.verify.sms.resendText', 'login')
+        : loc('oie.phone.verify.call.resendText', 'login');
+      const linkText = (this.model.get('mode') === 'sms')
+        ? loc('oie.resend.link', 'login')
+        : loc('oie.phone.verify.call.resendLinkText', 'login');
+      this.add(createCallout({
+        content: `${resendText}&nbsp;<a class='resend-link'>${linkText}</a>`,
+        type: 'warning',
+      }));
+    },
+
+    handleResendLink () {
+      this.options.appState.trigger('invokeAction', 'currentAuthenticatorEnrollment-resend');
+      // Hide warning, but start a timeout again..
+      if (!this.el.classList.contains('hide')) {
+        this.el.classList.add('hide');
+      }
+      this.showCalloutAfterTimeout();
+    },
+
+    postRender () {
+      this.showCalloutAfterTimeout();
+    },
+
+    showCalloutAfterTimeout () {
+      this.showCalloutTimer = setTimeout(() => {
+        this.el.classList.remove('hide');
+      }, SHOW_RESEND_TIMEOUT);
+    },
+
+    remove () {
+      View.prototype.remove.apply(this, arguments);
+      clearTimeout(this.showCalloutTimer);
+    }
+  },
+);
+
+const Body = BaseForm.extend(Object.assign(
+  {
+    className: 'phone-authenticator-challenge',
+    subtitle: ' ',
+
+    title () {
+      return loc('oie.phone.verify.title', 'login');
+    },
+
+    save () {
+      return loc('oie.verify.button', 'login');
+    },
+
+    initialize () {
+      BaseForm.prototype.initialize.apply(this, arguments);
+    },
+
+    render () {
+      BaseForm.prototype.render.apply(this, arguments);
+      const subtitleElement = this.el.querySelector('.okta-form-subtitle');
```

Comment:
`subtitle` can be dynamic as `save` button / `title`

Reply:
I need `html` in this case and courage doesn't support html for subtitle - https://github.com/okta/okta-signin-widget/blob/3e3c8f1870ae740846274460e3a0df916248c3e4/src/v2/view-builder/views/email/RequiredFactorEmailView.js#L70

Reply:
oh I see. I'm not fan of query class to do some override but at this case, maybe we could just add another view and style it. 
```
this.add(<div>...</div>)
```
not big deal though.

Reply:
I agree, that would be cleaner..